SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -474,3 +478,383 @@ void CFlowCompOutput::SetAdditionalScreenOutput(const CConfig *config){
 bool CFlowCompOutput::WriteHistoryFileOutput(const CConfig *config) {
   return !config->GetFinite_Difference_Mode() && COutput::WriteHistoryFileOutput(config);
 }
+
+void CFlowCompOutput::SetTurboPerformance_Output(std::shared_ptr<CTurboOutput> TurboPerf,
+                                  CConfig *config,
+                                  unsigned long TimeIter,
+                                  unsigned long OuterIter,
+                                  unsigned long InnerIter) {
+
+  curTimeIter  = TimeIter;
+  curAbsTimeIter = TimeIter - config->GetRestart_Iter();
+  curOuterIter = OuterIter;
+  curInnerIter = InnerIter;
+  stringstream TurboInOutTable, TurboPerfTable;
+
+  if(rank == MASTER_NODE) {
+    auto BladePerformance = TurboPerf->GetBladesPerformances();
+
+  /*-- Table for Turbomachinery Performance Values --*/
+  PrintingToolbox::CTablePrinter TurboInOut(&TurboInOutTable);
+
+    TurboInOutTable<<"-- Turbomachinery inlet and outlet property Summary:"<<endl;
+    TurboInOut.AddColumn("Properties", 25);
+    TurboInOut.AddColumn("Inlet", 25);
+    TurboInOut.AddColumn("Outlet", 25);
+    TurboInOut.SetAlign(PrintingToolbox::CTablePrinter::RIGHT);
+    TurboInOut.PrintHeader();
+
+    for (unsigned short iZone = 0; iZone <= config->GetnZone()-1; iZone++) {
+      auto nSpan = config->GetnSpan_iZones(iZone);
+
+      TurboInOut<<" BLADE ROW INDEX "<<iZone <<"";
+      TurboInOut.PrintFooter();
+      // TODO: Blade Wise Printing
+      TurboInOut << "Entropy "              << BladePerformance.at(iZone).at(nSpan)->GetInletState().GetEntropy()                     << BladePerformance.at(iZone).at(nSpan)->GetOutletState().GetEntropy();
+      TurboInOut << "Total Enthalpy "       << BladePerformance.at(iZone).at(nSpan)->GetInletState().GetTotalEnthalpy()               << BladePerformance.at(iZone).at(nSpan)->GetOutletState().GetTotalEnthalpy();
```

Comment:
formatting... also this repetition is so hard to read just do
```
const auto& BladePerf = BladePerformance.at(iZone).at(nSpan);
```
and then `BladePerf->Get...`